docs/platform-integration/essentials/contacts.md

```diff
@@ -1,61 +1,72 @@
+ms.date: 08/16/2021
 no-loc: ["Microsoft.Maui", "Microsoft.Maui.Essentials"]
 ---
 
 # Contacts
 
-The `Contacts` class lets a user pick a contact and retrieve information about it.
+This article describes how you can use the .NET Multi-platform App UI (.NET MAUI) Essentials `Contacts` class to select a contact and read information about it.
 
 ## Get started
 
 [!INCLUDE [get-started](includes/get-started.md)]
 
+[!INCLUDE [essentials-namespace](includes/essentials-namespace.md)]
+
 To access the **Contacts** functionality the following platform specific setup is required.
 
+<!-- markdownlint-disable MD025 -->
 # [Android](#tab/android)
 
 The `ReadContacts` permission is required and must be configured in the Android project. This can be added in the following ways:
 
-Open the **AssemblyInfo.cs** file under the **Properties** folder and add:
+- Add the assembly-based permission:
 
-```csharp
-[assembly: UsesPermission(Android.Manifest.Permission.ReadContacts)]
-```
+  Open the _AssemblyInfo.cs_ file under the **Properties** folder and add:
 
-OR Update Android Manifest:
+  ```csharp
+  [assembly: UsesPermission(Android.Manifest.Permission.ReadContacts)]
+  ```
 
-Open the **AndroidManifest.xml** file under the **Properties** folder and add the following inside of the **manifest** node.
+  \- or -
 
-```xml
-<uses-permission android:name="android.permission.READ_CONTACTS" /> />
-```
+- Update the Android Manifest:
+
+  Open the _AndroidManifest.xml_ file under the **Properties** folder and add the following inside of the `manifest` node:
+
+  ```xml
+  <uses-permission android:name="android.permission.READ_CONTACTS" /> />
+  ```
 
-Or right click on the Android project and open the project's properties. Under **Android Manifest** find the **Required permissions:** area and check this permission. This will automatically update the **AndroidManifest.xml** file.
+  \- or -
+
+- Use the Android project properties:
+
+  Right-click on the Android project and open the project's properties. Under **Android Manifest** find the **Required permissions:** area and check the **Contacts** permission. This will automatically update the **AndroidManifest.xml** file.
 
 # [iOS](#tab/ios)
 
-In your `Info.plist` add the following keys:
+In your _Info.plist_ file, add the following keys:
 
 ```xml
 <key>NSContactsUsageDescription</key>
 <string>This app needs access to contacts to pick a contact and get info.</string>
 ```
 
-Ensure that you update the `<string>` in the description that is specific for your app as it will be shown to your users.
+The `<string>` element is the description specific to your app and is shown to the user.
 
 # [Windows](#tab/windows)
 
-In the `Package.appxmanifest` under **Capabilities** ensure that `Contact` capability is checked.
+In the _Package.appxmanifest_, under **Capabilities**, ensure that the **Contact** capability is checked.
```

Comment:
You know, I have no idea if this will be valid in WinUI 3. Isn't package manifest a UWP thing?